ai-platform/notebooks/unofficial/tutorials/pytorch/sentiment_classification/pytorch-text-classification-caip-training.ipynb

```diff
@@ -0,0 +1,1170 @@
+      "metadata": {
+        "id": "rEJBSTyZIrIb"
+      },
+      "source": [
+        "# Training PyTorch Model on Google Cloud AI Platform Training \n",
+        "## Fine Tuning Pretrained [BERT](https://huggingface.co/bert-base-cased) Model for Sentiment Classification Task "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "X4cRE8IbIrIV"
+      },
+      "source": [
+        "# Overview\n",
+        "\n",
+        "This example is inspired from Token-Classification [notebook])(https://github.com/huggingface/notebooks/blob/master/examples/token_classification.ipynb) and [run_glue.py](https://github.com/huggingface/transformers/blob/master/examples/text-classification/run_glue.py) from HuggingFace 🤗. \n",
+        "We will be fine-tuning bert-base-cased (pre-trained) model.\n",
+        "You can find the details about this model at [🤗  Hub](https://huggingface.co/bert-base-cased).\n",
+        "\n",
+        "For more notebooks of the state of the art PyTorch/Tensorflow/JAX you can explore [🤗 Notebooks](https://huggingface.co/transformers/notebooks.html).\n",
+        "\n",
+        "### Dataset\n",
+        "\n",
+        "We will be using IMDB moview review dataset from Huggingface Datasets.\n",
+        "\n",
+        "### Objective\n",
+        "\n",
+        "Get familiar with PyTorch on Cloud AI Platform notebooks instances.\n",
+        "\n",
+        "### Costs \n",
+        "\n",
+        "This tutorial uses billable components of Google Cloud Platform (GCP):\n",
+        "\n",
+        "* Cloud AI Platform Notebook\n",
+        "* Cloud AI Platform Training\n",
+        "\n",
+        "Learn about [Cloud AI Platform\n",
+        "pricing](https://cloud.google.com/ml-engine/docs/pricing) and use the [Pricing\n",
+        "Calculator](https://cloud.google.com/products/calculator/)\n",
+        "to generate a cost estimate based on your projected usage."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cda8ab8b7a04"
+      },
+      "source": [
+        "## Setting up Notebook Environment\n",
+        "\n",
+        "This notebook assumes [PyTorch 1.7 DLVM](https://cloud.google.com/ai-platform/notebooks/docs/images) development environment. You can create a Notebook instance using [Google Cloud Console](https://cloud.google.com/ai-platform/notebooks/docs/create-new) or [`gcloud` command](https://cloud.google.com/sdk/gcloud/reference/notebooks/instances/create).\n",
+        "\n",
+        "```\n",
+        "gcloud notebooks instances create example-instance \\\n",
+        "    --vm-image-project=deeplearning-platform-release \\\n",
+        "    --vm-image-family=pytorch-1-7-cu110-notebooks \\\n",
+        "    --machine-type=n1-standard-4 \\\n",
+        "    --location=us-central1-a \\\n",
+        "    --boot-disk-size=100 \\\n",
+        "    --accelerator-core-count=1 \\\n",
+        "    --accelerator-type=NVIDIA_TESLA_T4 \\\n",
```

Comment:
You probably need some info about requesting T4 quota, as it's unlikely they'll have this out of the box.